Disambiguate Deed term (#166)
Meeds-io/whitepaper: en/4-xeeds.md

Our approach is based on decentralized and self-sovereign identity principles, allowing users to be in control at any time of what information they want to publicize, share with a restricted group, or keep private.

User's activity and engagement in the WoM is always linked to a community. A Xeed is the virtual persona of a user in a community. Xeed does not require any KYC, it is self-declaring, but we could conceivably imagine third-party firms providing this service.

## 4.2. Badges and credentials

> Badges and Credentials are given to users as a part of their participation in a community. The holder may download them as W3C Verifiable Credentials[^7][^8] (aka VCs) and store them in his wallet. Those VCs are proof of knowledge/know-how certified by the organization issuing it.

Author: this is not very clear, you should probably ...

In today’s world, it is hard to prove a skill beforehand. When people advertise themselves on networks, they count on their managers to give them a recommendation. The latter supposes certain goodwill on the part of the manager but is not necessarily treated as proof by the future employer.

The WoM provides a more transparent and immutable means for people to acquire and demonstrate their abilities.

Badges are used to represent accomplishments made within a community. After completing a set of objectives or challenges, users will be awarded points. Badges can be unlocked by accumulating certain amounts of points in specific categories.

When badges are downloaded by the user, it is possible to include the number of points and his rank in the Hub for this specific accomplishment.

> Credentials are more flexible than Badges; they could be a recommendation from a peer, indicating years of expertise, a training or exam completion, or engagement in the community.

Reviewer: ```suggestion
Credentials are more flexible than Badges; they could be a recommendation from a peer, indicating years of expertise, a training or exam completion, or engagement in the space.
```

> These are signed by the Hub and can be accompanied by a signature by a peer user.

Reviewer: ```suggestion
These are signed by the platform and can be accompanied by a signature by a peer user.
```

All of the above contributes to forming proven digital skillset that can be kept and used by the user when changing jobs or freelancing. It would be more trustworthy than traditional solutions as it relies on not just one individual but a full community to prove it.

Just like his Meeds wallet, the user will retain his Xeed, Badges, and Credentials attached to it in his identity wallet when moving from one Hub to another, from one city to another, anywhere through the WoM.

> The user will be able to display his Xeed, Badges, and Credentials on other Hubs to highlight his experience, as well as with recruiters or any other services that he deems interesting.

Reviewer: ```suggestion
The user will be able to express his Xeed, Badges, and Credentials towards other Hubs to highlight his experience, as well as with recruiters or any other services that he deems interesting.
```

The user will be able to select whether or not to reveal anything, what information to provide, and which identities to use for each position. The content (proven skill, previous Hubs, etc.) is tamper proof and so could not be modified. Any identity would have to be developed over time.

## 4.4. Fair work marketplace

Within this paradigm the future work marketplace allows users to move freely and allocate/sell their engagement among the Hubs – as they do in the physical world by changing employers. However, in the WoM their engagement would be tangible, measured and rewarded in a fair way. By moving between systems that follow a similar logic, they will better understand the rules of the game and will judge employers fairly – grass is not always greener elsewhere. Users will be able to see which Hub provides better engagement reward conditions, belongs to a more attractive city, etc.

> On their end, the Hub leaders can search for relevant skills using the WoM network of digital work identities (that the users decided to disclose). They will be able to source skills more effectively, based on verified and detailed information on work history, work skills and engagement level of the user.

Reviewer: ```suggestion
On their end, the community leaders can search for relevant skills using the WoM network of digital work identities (that the users decided to disclose). They will be able to source skills more effectively, based on verified and detailed information on work history, work skills and engagement level of the user.
```
